openupgradelib/openupgrade.py

```diff
@@ -26,11 +26,16 @@
 from openerp import release, tools, SUPERUSER_ID
 from openerp.tools.yaml_import import yaml_import
 from openerp.osv import orm
-from openerp.tools.mail import plaintext2html
 from openerp.modules.registry import RegistryManager
 import openerp.osv.fields
-import openerp.fields
 from . import openupgrade_tools
+if release >= "7.0":
+    from openerp.tools.mail import plaintext2html
+if release >= "8.0":
+    from openerp.fields.Many2many import Many2many, One2many
```

Comment:
They're 2 separates imports/logic: one is for a module/method only present for v7 and up, and the other is for new API declarations. It's working as I have tried them.